doc/man1/flux-shell.rst

```diff
@@ -332,30 +332,45 @@ plugins include:
   be set by the :man1:`flux-submit` and related commands :option:`--taskmap`
   option.
 
-.. option:: pmi=off
-
-  Disable the process management interface for parallel jobs (see below).
-
 .. option:: pmi=LIST
 
   Specify a comma-separated list of PMI implementations to enable.  If the
-  option is unspecified, the ``simple`` PMI-1 wire protocol implementation
-  is enabled.  Other options such as ``cray-pals`` or ``pmix`` may be
-  installed on your system.
+  option is unspecified, ``simple`` is assumed.  To disable, set *LIST* to
+  ``off``.  Available implementations include
+
+  simple
+    The simple PMI-1 wire protocol.  This implementation works by passing an
+    open file descriptor to clients via the :envvar:`PMI_FD` environment
+    variable.  It must be enabled when using Flux's ``libpmi.so`` and
+    ``libpmi2.so`` libraries are used, and is preferred by :man1:`flux-broker`
```

Comment:
Extra/unnecessary "using" in this sentence. (I assume you meant "It must be enabled when Flux's [...] libraries are used")